modules/EnsEMBL/Web/ConfigPacker.pm

```diff
@@ -35,80 +31,46 @@ sub munge_config_tree {
-         from meta_coord'
-    );
-    foreach my $row ( @$t_aref ) {
-      $self->db_details($db_name)->{'tables'}{$row->[0]}{'coord_systems'}{$row->[1]}=$row->[2];
-    }
-  }
-#---------- Meta table (everything except patches)
-## Needs tweaking to work with new ensembl_ontology_xx db, which has no species_id in meta table
-  if( $self->_table_exists( $db_name, 'meta' ) ) {
-    my $hash = {};
-
-## EG MULTI
-# With multi species DB there is no way to define the list of chromosomes for the karyotype in the ini file
-# The idea is the people who produce the DB can define the lists in the meta table using region.toplevel met key
-# In case there is no such definition of the karyotype - we just create the lists of toplevel regions 
-    if($db_name =~ /CORE/) {
-      if ($self->is_collection('DATABASE_CORE')) {
-        my $t_aref = $dbh->selectall_arrayref(
-          qq{SELECT cs.species_id, s.name FROM seq_region s, coord_system cs
-          WHERE s.coord_system_id = cs.coord_system_id AND cs.attrib = 'default_version' AND cs.name IN ('plasmid', 'chromosome')
-          ORDER BY cs.species_id, s.name, s.seq_region_id}
-        );
-
-        foreach my $row ( @$t_aref ) {
-            push @{$hash->{$row->[0]}{'region.toplevel'}}, $row->[1];
-        }
-     }
-   }
-##
-    $t_aref  = $dbh->selectall_arrayref(
-      'select meta_key,meta_value,meta_id, species_id
-         from meta
-        where meta_key != "patch"
-        order by meta_key, meta_id'
-    );
-
-    foreach my $r( @$t_aref) {
-      push @{ $hash->{$r->[3]+0}{$r->[0]}}, $r->[1];
-    }
-    
-    $self->db_details($db_name)->{'meta_info'} = $hash;
-  }
+sub _intraspecies_sql {
+## need to exclude HOMOEOLOGUES as well as PARALOGUES otherwise too many method link species sets that prevents web site from starting
+  return qq(
+    select mls.species_set_id, mls.method_link_species_set_id, count(*) as count
+      from method_link_species_set as mls, 
+        method_link as ml, species_set as ss, genome_db as gd 
+      where mls.species_set_id = ss.species_set_id
+        and ss.genome_db_id = gd.genome_db_id 
+        and mls.method_link_id = ml.method_link_id
+        and ml.type not like "%PARALOGUES"
+        and ml.type not like "%HOMOEOLOGUES"
+        and mls.source != "ucsc"
+      group by mls.method_link_species_set_id, mls.method_link_id
+      having count = 1
+  );
 }
 
-## EG : need to exclude HOMOEOLOGUES as well as PARALOGUES otherwise too many method link species sets that prevents web site from starting
-sub _summarise_compara_db {
-  my ($self, $code, $db_name) = @_;
-  
-  my $dbh = $self->db_connect($db_name);
-  return unless $dbh;
-  
-  push @{$self->db_tree->{'compara_like_databases'}}, $db_name;
+sub _homologies_sql {
+## need to exclude HOMOEOLOGUES as well as PARALOGUES otherwise too many method link species sets that prevents web site from starting
+  return qq(
+    select ml.type, gd.name, gd.name, count(*) as count
```

Comment:
same here we two gd.name, i guess we can remove one of them unless it is supposed to be another column